### DISCUSSION TITLE
Other ways to trigger "F"

### DISCUSSION THREAD

**Discussion by PraveenKumarpkm** [gdh1995/vimium-c#923] [Q&A]
I'm wondering if I can trigger the "f" key in vimium c. 

I already have the "f" key working but in some situations because of typing it doesn't work and obviously ignores the "f" key.

I know that the escape key could work but I wonder if I can set one more shortcut to trigger the "F" key if so how?

I wanna keep the "f" shortcut as well the new shortcut

**Comment by gdh1995** [comment]
You may map other keys to `LinkHints.activate***`, as said in https://github.com/gdh1995/vimium-c#custom-key-mappings

**Comment by wetlife** [comment]
If we are typing when we display links then how can we follow links?
